### PR TITLE
ELEMENTS-993: share-button improvements to keep consistency with easyshare

### PR DESCRIPTION
**Pull request by nmpcunha**

### CODE REVIEW

File: ui/actions/nuxeo-share-button.js [hunk 1]
```diff
@@ -136,28 +148,39 @@ import './nuxeo-action-button-styles.js';
...
+
+      this._selectLink(link);
       if (!window.document.execCommand('copy')) {
         return;
       }
 
-      const link = this.$.copyLink;
-      this._debouncer = Debouncer.debounce(this._debouncer, timeOut.after(3000), () => {
-        link.innerText = this.i18n('shareButton.operation.copy');
+      button._debouncer = Debouncer.debounce(button._debouncer, timeOut.after(2000), () => {
```
Comment (Gabez0r) [inline ui/actions/nuxeo-share-button.js]: Do we need to change the timeout value here?

File: ui/actions/nuxeo-share-button.js [hunk 2]
```diff
@@ -136,28 +148,39 @@ import './nuxeo-action-button-styles.js';
...
+        button.classList.remove('selected');
       });
-      link.innerText = this.i18n('shareButton.operation.copied');
+
+      button.set('icon', 'check');
+      button.classList.add('selected');
+      this.fire('notify', { message: this.i18n('shareButton.operation.copied'), duration: 2000 });
+    }
+
+    _selectLink(link) {
```
Comment (Gabez0r) [inline ui/actions/nuxeo-share-button.js]: No need to extract a function that is only used once? (and in this case, only has a single line of code)

File: ui/actions/nuxeo-share-button.js [hunk 3]
```diff
@@ -136,28 +148,39 @@ import './nuxeo-action-button-styles.js';
...
+      link.$.paperInput.$.nativeInput.select();
     }
 
-    _selectPermalink() {
-      this.$.permalink.$.paperInput.inputElement.inputElement.select();
+    _unselectLink(link) {
```
Comment (Gabez0r) [inline ui/actions/nuxeo-share-button.js]: Same here.

File: ui/actions/nuxeo-share-button.js (hunk 1 above)
Comment (nmpcunha) [reply ui/actions/nuxeo-share-button.js]: It was suggested by Carolina to reduce the time because three seconds seems a lot.

**Pull request merged**